nixos/tests/fcitx5/default.nix

```diff
@@ -94,22 +94,33 @@ rec {
             machine.sleep(1)
 
             ### Default wubi, enter 一下
-            machine.send_chars("gggh")
+            machine.send_chars("gggh ")
```

Comment:
Why is a space needed here? For wubi, gggh can get the characters automatically because four codes are input and only one candidate for these code.

Reply:
I did some manual interactive testing, and it seems like there are two candidates for `gggh`:
![image](https://github.com/NixOS/nixpkgs/assets/7698349/cf6245fc-c98e-423e-b528-8ea3292b8311)